content/blog/2022-04-14-april-22-heartbeat.md

```diff
@@ -0,0 +1,339 @@
+---
+title: April '22 Heartbeat
+date: 2022-04-14
+description: >
+  Monthly updates are here! You will find the future of AI Infrastruture is
+  modular, articles on distribution drift and how to solve it, the usual great
+  tutorials and workflows from the Community,  online course updates, new docs
+  and more! Happy April!
+
+descriptionLong: |
+  This month you will find:
+
+    🧱 AI Infrastructure is becoming modular,
+
+    🔎 Distribution drift with Chip Huyen
+    
+    🥰 Tutorials and workflows from the Community,
+
+    🗣 IRL Events,
+
+    💻 Online Course(s) updates,
+
+    🚀 Quebec becoming an Iterative Hub, and more!
```

Comment:
Other than names or companies, all of the words after the first word on a line should start with lower case.